### ISSUE TITLE
Back Button

### ISSUE THREAD

**Issue by LlamaPlays** [Flieh/diplicity-beta-feedback#7]
I suggest adding a back button, to help the game become more accessible. Right now, using the arrow on my device seems rather inconvenient, seeing as many people, aren't used to not having a back button in the game. It takes a while to get that 'muscle memory' out of the mind, so I suggest it, getting rid of the hassle to new players (and old).

**Comment by zond** [comment]
I really prefer using the Android way, and not as extra buttons where the back button already exists on all Android phones

**Comment by Flieh** [comment]
I'm cool with the stock android button too, except it would be nice to prevent it from exiting the app. Maybe force confirmation. I've seen many apps that do it this way.

**Comment by zond** [comment]
Is it a problem that it closes the app? It's the way I tend to close apps :)

**Comment by LlamaPlays** [comment]
I don't close apps that way, but it is frustrating if I tap back more than twice and it goes out. I think force confirmation would solve the problem.

**Comment by Flieh** [comment]
I agree with llama on this. I use the home button to close apps. There are apps that let you disable the back button for app exit as an option which could accommodate both types of users.

**Comment by Flieh** [comment]
Droidippy only exits via back button if you're on the game list page. This works. I guess once the user has the feel for when he's at the root page, the problem goes away. Beta testers just aren't familiar with the interface yet so we don't always know where we are.

**Comment by Flieh** [comment]
I guess this ties in with Tommy's issue about the initial landing page. If you're on your list of open games, you'll know you can back button out of the app and otherwise it gets you to where you were.

**Comment by zond** [comment]
I agree it's a non trivial problem. Let's key it stew a while? :)

**Comment by Flieh** [comment]
Yes
